### PR TITLE
[TASK] Add PHP CS Fixer to GitLab

### PR DESCRIPTION
**Pull request by simonschaufi**
I've just noticed that PHP CS Fixer is only executed on GitHub but not on GitLab

### CODE REVIEW

**Comment by simonschaufi** [comment]
I've just noticed that PHP CS Fixer is only executed on GitHub but not on GitLab

**Review by oliverklee — APPROVED** [review]
Thanks, looks good!

I'd like to wait for the build to appear on https://git.typo3.org/qa/example-extension/-/pipelines so we can see this in action before merging.

**Comment by oliverklee** [comment]
Ah, we won't see this on GitLab for fork. Then I'll merge right now.

**Pull request merged**
